packages/default-theme/components/SwGenericMethod.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <div class="sw-generic-method">
+    <h4 class="sw-generic-method__title">{{ label }}</h4>
+    <p class="sw-generic-method__content">
+      {{ method.name }}
+    </p>
+  </div>
+</template>
+
+<script>
+import { SfInput, SfButton, SfAlert, SfTable } from '@storefront-ui/vue'
+
+import SwPluginSlot from 'sw-plugins/SwPluginSlot'
```

Comment:
these imports are not used in component

also, please rename to `SwCheckoutMethod`